operatorconfig/moduleconfig/common/version-values.yaml

```diff
@@ -32,6 +32,8 @@ powerstore:
   # List of Driver versions and modules that supports the version
   v2.7.0:
       resiliency: "v1.6.0"
+  v2.8.0:
+      resiliency: "v1.7.0"
```

Comment:
@suryagupta4 Is it finalized that we will be releasing "v1.7.0" of resiliency module?